examples/notify-slack-simple/versions.tf

```
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.61"
+      version = ">= 4.0.0"
```

```suggestion
      version = ">= 4.8"
```
4.8 or newer is required by Lambda module. https://github.com/terraform-aws-modules/terraform-aws-lambda/blob/v3.1.0/versions.tf#L7